client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -21,32 +22,167 @@ const EditProfilePage = () => {
   const fileInputRef = useRef<HTMLInputElement>(null);
 
   const [formData, setFormData] = useState({
+    cohort: "",
     email: "",
+    firstName: "",
+    linkedInProfile: "",
+    gitHubProfile: "",
+    nickName: "",
     personalBio: "",
+    skills: [] as string[],
+    specializations: [] as string[],
+    careerInformation: {
+      currentPosition: {
+        title: "",
+        company: "",
+      },
+    },
+    socialMediaLinks: {
+      twitter: "",
+      blog: "",
+    },
+    availabilityForNetworking: false,
   });
 
+ 
+
   const [file, setFile] = useState<File | null>(null);
 
   useEffect(() => {
     if (userID) dispatch(fetchUserProfile(userID as string));
   }, [dispatch]);
```

Comment:
add userId to array

Reply:
Hey @JimmyTran48, thanks for your feedback! I'm not entirely sure I understand your comment. Could you please provide more detail or specify what you mean?